#!/usr/bin/env python
PACKAGE = "simulator"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("velocity",          double_t,    0, "velocity, pixels/s",       0, 0, 1000)
gen.add("max_obstacle_range",double_t,    0, "max range of sensors",     25, 0, 500)
gen.add("show_path",         bool_t,      0, "trace path",               False)
gen.add("show_scan",         bool_t,      0, "show laser scan",          False)
gen.add("video_filepath",    str_t,       0, "location to save movie",   "")
gen.add("record_video",      bool_t,      0, "enable video recording",   False)
gen.add("reset_sim",         bool_t,      0, "reset simulation",         False)

exit(gen.generate(PACKAGE, "simulator", "Simulator"))
